### DISCUSSION TITLE
[Question and support]

### DISCUSSION THREAD

**Discussion by renussunre** [PrivateBin/PrivateBin#1152] [Q&A] [answered]
### Describe the problem/question

Hi, I have a PrivateBin 1.5.2 with Apache2

I've changed the upload_max_filesize, post_max_size and max_execution_time, in the php.ini of apache2. 
And i've tried other things, like using msyql. 
The fact is that I wanted to share files with my own server without using Cloud's. But I only can post/get files of 100Mb. 
When I try with 200mb fails to download. The web gets stuck with ''Decryption paste''
<img width="251" alt="image" src="https://github.com/PrivateBin/PrivateBin/assets/142978303/9dfdfb9a-2d69-40b8-84ec-032a9a814369">

And if I look into the log, I only see that ''works fine''.
<img width="661" alt="image" src="https://github.com/PrivateBin/PrivateBin/assets/142978303/6566b7c0-cd76-4b85-b159-08149b23737c">

¿Wich limit size have you achieve with these project? ¿Is there a way for doing that? I know that the project is only for lower size, but I wanted to implement for a private share files.

### Did you use the FAQ section?

- [X] Yes, I have read the FAQ and I found no solution/answer there.

### What you did?

1.
2.
...

### PrivateBin version

1.5.2

### Issue reproducibility

Yes

**Comment by rugk** [comment, marked as answer]
Duplicate of https://github.com/orgs/PrivateBin/discussions/1153

You already submitted this. Please do not duplicate discussions.
